Upstream-Status: Inappropriate [configuration]

The AC_LANG sections upset libtool 2.2.2, as do .cpp files without a call
to AC_PROG_CXX. Easiest solution is to patch this out for now.

RP - 14/4/08 

Index: hal-0.5.9.1/configure.in
===================================================================
--- hal-0.5.9.1.orig/configure.in	2008-04-14 22:41:49.000000000 +0100
+++ hal-0.5.9.1/configure.in	2008-04-14 22:43:13.000000000 +0100
@@ -345,32 +345,8 @@
 fi
 
 dnl Check for libsmbios
-AC_LANG_PUSH([C++])
-AC_CHECK_LIB(smbios, SMBIOSFreeMemory, LIB_SMBIOS=yes , LIB_SMBIOS=no )
-AC_LANG_POP([C++])
-if test "$LIB_SMBIOS" = "yes" ; then
-   AC_MSG_CHECKING([for libsmbios >= 0.13.4])
-   AC_TRY_RUN(
-   #include <smbios/version.h>
-   int main ()
-   {
-        int major ;
-        int minor ;
-        int micro ;
-
-        if ( sscanf( LIBSMBIOS_RELEASE_VERSION , "%d.%d.%d", &major, &minor, &micro ) == 3 ) {
-	    if ((major == 0 && minor == 13 && micro >= 4) ||
-               (major >= 0 && minor > 13)) {
-		return 0;
-	    }
-	}
-	return 1;
-   } , [USE_SMBIOS=yes; AC_MSG_RESULT(yes); AM_CONDITIONAL(HAVE_SMBIOS,true)],
-       [USE_SMBIOS=no; AC_MSG_RESULT(failed); AM_CONDITIONAL(HAVE_SMBIOS,false)])
-else
-  USE_SMBIOS=no
-  AM_CONDITIONAL(HAVE_SMBIOS,false)
-fi
+USE_SMBIOS=no
+AM_CONDITIONAL(HAVE_SMBIOS,false)
 
 AC_ARG_WITH([libpci],
   [AS_HELP_STRING([--without-libpci],
Index: hal-0.5.9.1/hald/linux/addons/Makefile.am
===================================================================
--- hal-0.5.9.1.orig/hald/linux/addons/Makefile.am	2008-04-14 22:45:18.000000000 +0100
+++ hal-0.5.9.1/hald/linux/addons/Makefile.am	2008-04-14 22:46:05.000000000 +0100
@@ -60,11 +60,6 @@
 hald_addon_usb_csr_LDADD = $(top_builddir)/libhal/libhal.la -lusb @GLIB_LIBS@
 endif
 
-if BUILD_DELL
-libexec_PROGRAMS += hald-addon-dell-backlight
-hald_addon_dell_backlight_SOURCES = addon-dell-backlight.cpp ../../logger.c 
-hald_addon_dell_backlight_LDADD = $(top_builddir)/libhal/libhal.la -lsmbios @GLIB_LIBS@ 
-endif
 endif
 
 hald_addon_acpi_SOURCES = addon-acpi.c ../../logger.c ../../util_helper.c
